integrations/acquisition/covid_hosp/state_daily/test_scenarios.py

```diff
@@ -11,39 +11,30 @@
 
 # first party
 from delphi.epidata.acquisition.covid_hosp.state_daily.database import Database
-from delphi.epidata.acquisition.covid_hosp.common.test_utils import UnitTestUtils
+from delphi.epidata.acquisition.covid_hosp.common.test_utils import CovidHospTestCase, UnitTestUtils
 from delphi.epidata.client.delphi_epidata import Epidata
-from delphi.epidata.acquisition.covid_hosp.state_daily.update import Update
-from delphi.epidata.acquisition.covid_hosp.state_daily.network import Network
-from delphi.epidata.acquisition.covid_hosp.common.utils import Utils
-import delphi.operations.secrets as secrets
+from delphi.epidata.acquisition.covid_hosp.common.network import Network
+from delphi.epidata.common.covid_hosp.covid_hosp_schema_io import CovidHospSomething
 
 # py3tester coverage target (equivalent to `import *`)
 __test_target__ = \
     'delphi.epidata.acquisition.covid_hosp.state_daily.update'
 
 
-class AcquisitionTests(unittest.TestCase):
+class AcquisitionTests(CovidHospTestCase):
 
   def setUp(self):
     """Perform per-test setup."""
 
     # configure test data
     self.test_utils = UnitTestUtils(__file__)
 
-    # use the local instance of the Epidata API
-    Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
-
-    # use the local instance of the epidata database
-    secrets.db.host = 'delphi_database_epidata'
-    secrets.db.epi = ('user', 'pass')
-
     # clear relevant tables
-    with Database.connect() as db:
-      with db.new_cursor() as cur:
-        cur.execute('truncate table covid_hosp_state_daily')
-        cur.execute('truncate table covid_hosp_state_timeseries')
-        cur.execute('truncate table covid_hosp_meta')
+    # TODO: get these names dynamically from the YAML file once all are available
+    super().setUp(Database(), [
+      'covid_hosp_state_daily',
+      'covid_hosp_meta'
+    ])
```

Comment:
with my other suggested changes to the base `CovidHospTestCase`, all of this `setUp()` method can be replaced with just:
```
  db_class = Database
  test_util_context = __file__
```
thereby getting `super().setup()` for free